package.json

```diff
@@ -37,16 +37,27 @@
 		"defaults"
 	],
 	"dependencies": {
-		"@wordpress/dom-ready": "^3.3.1",
+		"@wordpress/dom-ready": "^3.4.1",
 		"js-cookie": "^3.0.1"
 	},
 	"devDependencies": {
+		"@wordpress/api-fetch": "^6.1.1",
 		"@wordpress/babel-preset-default": "^6.6.1",
+		"@wordpress/block-editor": "^8.3.1",
+		"@wordpress/blocks": "^11.3.1",
+		"@wordpress/components": "^19.6.1",
+		"@wordpress/compose": "^5.2.1",
+		"@wordpress/dom-ready": "^3.4.1",
 		"@wordpress/e2e-test-utils": "^7.0.1",
+		"@wordpress/editor": "^12.3.1",
```

Comment:
I'm not sure we're using this dependency. We could remove it.

Reply:
We definitely need those packages. They are not used in the code itself but in our tooling, so I guess that's why they are being flagged as not used.

Reply:
👍 Marking this as resolved. If anything arises we'll revert.